src/main/java/teammates/ui/controller/InstructorFeedbacksPageData.java

```diff
@@ -270,15 +284,18 @@ public FeedbackSessionsCopyFromModal getCopyFromModal() {
             }
         }
         
-        // Add "No active courses" option if there are no active courses
-        if (result.isEmpty()) {
-            ElementTag blankOption = createOption("No active courses!", "", true);
-            result.add(blankOption);
-        }
-        
         return result;
     }
-
+    
+    /**
+     * Adds the default option to the list of select options if the list is empty.
+     * @param selectOptions list containing all the options
+     */
+    private void addDefaultOption(List<ElementTag> selectOptions) {
+        ElementTag defaultOption = createOption("No active courses!", "", true);
+        selectOptions.add(defaultOption);
+    }
```

Comment:
Try changing the "default" term used here.
"default" doesn't seem very suitable as it somewhat implies that it's an actual option.
Maybe something like "placeholder" instead?